pkg/crd/crd_test.go

```diff
@@ -0,0 +1,308 @@
+// Package crd helps with the installation of CustomResourceDefinitions.
```

Comment:
Could you please update this package comment as well? This one seems to win when it comes to our godoc - there are two package comments now.

Reply:
missed this one.